### DISCUSSION TITLE
SSO and Github userID

### DISCUSSION THREAD

**Discussion by jmaricq** [community/community#52105] [Enterprise]
### Select Topic Area

Product Feedback

### Body

We are forcing SSO to connect to our internal organisation. Users need to link their Github Account to our internal account to have access to our Github organisation. That organisation is only accessible for our internal users, connecting with SSO.

My problem is I don't have any possibility to see the "Display Name" nor the userID coming from our IdP when connecting with SSO. I only see the Github userID but it is very complicate to invite someone to repo or find back my colleagues in the jungle of pseudo.

Is there a way that Github integrate an option (like a toggle button that organisation admin could opt-in) to force the usage of the IdP display name for the users instead of the github userID when SSO is enforced on an organisation level?

Thanks for the feedback!

**Comment by petkoaa** [comment]
Hi, you might want to look into this page: https://docs.github.com/en/enterprise-cloud@latest/organizations/granting-access-to-your-organization-with-saml-single-sign-on/viewing-and-managing-a-members-saml-access-to-your-organization

If you found some other way please elaborate since im trying to do a similar thing but with the API
